setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = netbox-slm
-version = 1.4
+version = 1.5
```

Comment:
Usually the version is bumped where there's a collection of changes to be shipped

Reply:
I will remove it with the next commit